app/src/main/java/com/github/alfabravo2013/readyforexams/ReadyForExamApp.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.alfabravo2013.readyforexams
+
+import android.app.Application
+import com.github.alfabravo2013.readyforexams.di.appModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
+
+class ReadyForExamApp : Application() {
```

Comment:
I would suggest MainApplication as naming but you can leave this too. Your choice.